cms/earn/strategies.json

```diff
@@ -2639,7 +2719,119 @@
           "_comment": "USDC"
         }
       ],
-      "categories": ["Correlated", "Blue Chip", "Stablecoins"]
+      "categories": [
+        "Correlated",
+        "Blue Chip",
+        "Stablecoins"
+      ]
+    },
+    {
+      "id": "osmo15fqmdl8lfl9h0qflljd63ufw9j2m7xmsk3hu5vsn8xpta4hk5chqt7mddc",
+      "name": "OSMO Redemption Rate Arbitrage",
+      "platform": "Margined",
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/osmo-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=1475100361#gid=1475100361",
+      "startDateTimeUtc": "2024-07-21T00:00:00Z",
+      "unlisted": false,
+      "disabled": false,
+      "message": "Buys Liquid Staked Assets at a discount and performs the unbonding to earn the delta.",
+      "depositDenoms": [
+        {
+          "coinMinimalDenom": "uosmo",
+          "_comment": "Osmosis"
+        }
+      ],
+      "positionDenoms": [
+        {
+          "coinMinimalDenom": "factory/osmo15fqmdl8lfl9h0qflljd63ufw9j2m7xmsk3hu5vsn8xpta4hk5chqt7mddc/redemption-vault-uosmo",
+          "_comment": "OSMO Locust Redemption Vault Token"
+        }
+      ],
+      "rewardDenoms": [],
+      "categories": [
+        "Correlated",
+        "Blue Chip"
+      ]
+    },
+    {
+      "id": "osmo1reyz7pwu7y9e7lmzqg6j4h7jcv32du7n7jhnk2lz93a9lxr56ess2qtgzl",
+      "name": "TIA Redemption Rate Arbitrage",
+      "platform": "Margined",
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/tia-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=1058195732#gid=1058195732",
```

Comment:
Please replace with this URL: https://docs.google.com/spreadsheets/d/1RlH1KT6EUJCY6ngJ_8TdPp9csF9T9iB8euyNlGSM17Y/edit?gid=1503172972#gid=1503172972